packages/stream/common/utils.ts

```diff
@@ -30,6 +30,21 @@ export const getNumberFromBN = (value: BN, decimals: number): number =>
     ? value.div(new BN(10 ** decimals)).toNumber()
     : value.toNumber() / 10 ** decimals;
 
+/**
+ * Calculate total amount of a Contract including all fees.
+ * - first we convert fee floating to a BN with up to 4 decimals precision
+ * - then we reverse the fee with `feeMultiplier` to safely multiply it by depositedAmount
+ *   to receive a total number and not percentage of depositedAmount
+ * @param depositedAmount deposited raw tokens
+ * @param totalFee sum of all fees in percentage as floating number, e.g. 0.99% should be supplied as 0.99
+ * @returns total tokens amount that Contract will retrieve from the Sender
+ */
+export const calculateTotalAmountToDeposit = (depositedAmount: BN, totalFee: number): BN => {
```

Comment:
and I think that FE should migrate to this for alignment with sdk, bc we have this calculation in two places and it can slip

btw why it is much easier than our impl in the app 😄 https://github.com/streamflow-finance/streamflow-app/blob/master/src/pages/CreateStreamPage/CreateStreamReview.tsx#L111